lib/SILOptimizer/Mandatory/SendNonSendable.cpp

```diff
@@ -857,6 +914,16 @@ class PartitionOpTranslator {
     case SILInstructionKind::SuperMethodInst:
     case SILInstructionKind::ObjCSuperMethodInst:
       return translateSILAssign(inst->getResult(0), inst->getOperand(0));
+    case SILInstructionKind::BeginBorrowInst: {
+      return translateSILAssign(inst->getResult(0), inst->getOperand(0));
+    }
+
+    case SILInstructionKind::EnumInst: {
```

Comment:
Is this case new? Does it have a test?

Reply:
Yes. It came up with the enum test cases I added below. I wish we had SIL test cases which would make it easier to understand the connection.